traceloop.go

```diff
@@ -10,9 +10,26 @@ import (
 	"strconv"
 	"time"
 
+	log "github.com/sirupsen/logrus"
+
 	"github.com/kinvolk/traceloop/pkg/straceback"
 )
 
+func init() {
+	lvl, ok := os.LookupEnv("LOG_LEVEL")
+	if !ok {
+		// LOG_LEVEL not set, let's default to info
+		lvl = "info"
+	}
+	// parse string
+	ll, err := log.ParseLevel(lvl)
+	if err != nil {
+		ll = log.DebugLevel
```

Comment:
Why if the log level is not set it defaults to info but if the log level is set to a wrong value it defaults to debug?

Reply:
It could be a good idea to print a log indicating the env variable is wrong and the implementation is defaulting to ...

Reply:
I am changing it to a flag, and it errors out if it is wrong.